.gitignore

```diff
@@ -1,7 +1,6 @@
 .DS_Store
 .sass-cache
 node_modules
-**/package-lock.json
```

Comment:
![image](https://user-images.githubusercontent.com/54012/39939853-6882fd6c-550c-11e8-947c-1bdda63b5e61.png)
Should we direct that at `modules/` folders?